python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
In fact this change has caused some big problems since it's setting the parent of returned object to be `self` which is a ` Descriptor object. I will have to revert or change this as it's breaking 6.1.15.